app/components/admin/Hooks/Form.vue

```diff
@@ -83,8 +83,10 @@ defineExpose({
       type-override="url"
       v-model="url"
     />
-    <VInputButton :loading="changing" type="submit">
-      {{ isUpdating ? 'Save' : 'Create' }}
-    </VInputButton>
+    <div class="w-full flex justify-center">
```

Comment:
```suggestion
    <div class="w-full flex justify-center">
```

@amandesai01 you mean don't wrap in `div` instead change in `VInputButton` model ?

Reply:
Oh, we can fix it in form itsef just relise, also looking cool


![image](https://github.com/user-attachments/assets/d5db88a0-a104-4b59-9a20-05f7279bcd64)